jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/kubernetes/DockerBuildServiceTest.java

```diff
@@ -13,78 +13,76 @@
  */
 package org.eclipse.jkube.kit.config.service.kubernetes;
 
-import mockit.Expectations;
-import mockit.Verifications;
-import org.eclipse.jkube.kit.build.service.docker.BuildService;
-import org.eclipse.jkube.kit.common.JKubeConfiguration;
 import org.eclipse.jkube.kit.config.image.build.BuildConfiguration;
 import org.eclipse.jkube.kit.config.image.ImageConfiguration;
-import org.eclipse.jkube.kit.build.service.docker.ImagePullManager;
-import mockit.Mocked;
-import mockit.VerificationsInOrder;
+import org.eclipse.jkube.kit.build.service.docker.BuildService;
 import org.eclipse.jkube.kit.config.service.JKubeServiceException;
 import org.eclipse.jkube.kit.config.service.JKubeServiceHub;
+import org.junit.Before;
 import org.junit.Test;
 
 import java.io.IOException;
 import java.util.Collections;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.isNull;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 public class DockerBuildServiceTest {
 
-  @Mocked
-  private JKubeServiceHub jKubeServiceHub;
+  private JKubeServiceHub mockedJKubeServiceHub;
+  private ImageConfiguration image;
+  private BuildService mockedDockerBuildService;
 
-  @Test
-  public void build_withValidConfiguration_shouldBuildAndTag() throws Exception {
-    // Given
-    final ImageConfiguration image = ImageConfiguration.builder()
+  @Before
+  public void setUp() {
+    mockedJKubeServiceHub = mock(JKubeServiceHub.class, RETURNS_DEEP_STUBS);
+    mockedDockerBuildService = mock(BuildService.class, RETURNS_DEEP_STUBS);
```

Comment:
I think it's more readable if you skip these lines and 
- add `@Mock(answer = Answers.RETURNS_DEEP_STUBS)` to the 2 attributes
- use runner `@RunWith(MockitoJUnitRunner.class)`

WDYT ?

Reply:
okay, let me update this as per your suggestion.